trying to compile latest git on freebsd 12.2 error missing ;

autogen runs
configure runs
make comes with this error:

line 1247 missing `;`

```
# make
make  all-recursive
Making all in messages
Making all in src
Making all in libltfs
Making all in tape_drivers/generic/file
Making all in tape_drivers/generic/itdtimg
Making all in tape_drivers/freebsd/cam
/bin/sh ../../../../libtool  --tag=CC --tag=disable-static  --mode=compile cc -DHAVE_CONFIG_H -I. -I../../../..  -D_GNU_SOURCE -I../../../../src -DLTFS_CONFIG_FILE='"/usr/local/etc/ltfs.conf"' -DLTFS_BASE_DIR='"/usr/local"' -DUSE_NEW_LOCKING -I ../../.. -I ../..  -Wall -Wsign-compare -fsigned-char -I/usr/local/include/fuse -D_FILE_OFFSET_BITS=64 -I/usr/local/include/uuid -I/usr/local/include -I/usr/local/include/libxml2 -I/usr/local/include -D ENABLE_SNMP -O2 -pipe -I/usr/local/include -I/include -fstack-protector-strong -fno-strict-aliasing -std=c99 -DNETSNMP_ENABLE_IPV6 -fno-strict-aliasing -DNETSNMP_REMOVE_U64 -O2 -pipe -I/usr/local/include -I/include -fstack-protector-strong -fno-strict-aliasing -std=c99 -Ufreebsd12 -Dfreebsd12=freebsd12 -DHAS_FPSETMASK -DHAS_FLOATINGPOINT_H -DUSE_THREAD_SAFE_LOCALE -fno-strict-aliasing -pipe -fstack-protector-strong -I/usr/local/include -I/usr/local/lib/perl5/5.32/mach/CORE -I/usr/include -I. -I/usr/local/include -g -O2 -D_FORTIFY_SOURCE=0 -O0 -ggdb -MT libtape_cam_la-cam_tc.lo -MD -MP -MF .deps/libtape_cam_la-cam_tc.Tpo -c -o libtape_cam_la-cam_tc.lo `test -f 'cam_tc.c' || echo './'`cam_tc.c
libtool: compile:  cc -DHAVE_CONFIG_H -I. -I../../../.. -D_GNU_SOURCE -I../../../../src -DLTFS_CONFIG_FILE=\"/usr/local/etc/ltfs.conf\" -DLTFS_BASE_DIR=\"/usr/local\" -DUSE_NEW_LOCKING -I ../../.. -I ../.. -Wall -Wsign-compare -fsigned-char -I/usr/local/include/fuse -D_FILE_OFFSET_BITS=64 -I/usr/local/include/uuid -I/usr/local/include -I/usr/local/include/libxml2 -I/usr/local/include -D ENABLE_SNMP -O2 -pipe -I/usr/local/include -I/include -fstack-protector-strong -fno-strict-aliasing -std=c99 -DNETSNMP_ENABLE_IPV6 -fno-strict-aliasing -DNETSNMP_REMOVE_U64 -O2 -pipe -I/usr/local/include -I/include -fstack-protector-strong -fno-strict-aliasing -std=c99 -Ufreebsd12 -Dfreebsd12=freebsd12 -DHAS_FPSETMASK -DHAS_FLOATINGPOINT_H -DUSE_THREAD_SAFE_LOCALE -fno-strict-aliasing -pipe -fstack-protector-strong -I/usr/local/include -I/usr/local/lib/perl5/5.32/mach/CORE -I/usr/include -I. -I/usr/local/include -g -O2 -D_FORTIFY_SOURCE=0 -O0 -ggdb -MT libtape_cam_la-cam_tc.lo -MD -MP -MF .deps/libtape_cam_la-cam_tc.Tpo -c cam_tc.c  -fPIC -DPIC -o .libs/libtape_cam_la-cam_tc.o
cam_tc.c:1247:50: error: expected ';' after expression
                pos->block = scsi_8btou64(ext_data.last_object)
                                                               ;
1 error generated.
*** Error code 1

Stop.
make[3]: stopped in /root/ltfs/src/tape_drivers/freebsd/cam
*** Error code 1
*** Error code 1
*** Error code 1
root@jails:~/ltfs # grep . -R -e 'scsi_8btou64'
./src/tape_drivers/freebsd/cam/cam_tc.c:                pos->block = scsi_8btou64(ext_data.last_object)
```

```
root@jails:~/ltfs # vi src/tape_drivers/freebsd/cam/cam_tc.c
   1242
   1243         if (rc != DEVICE_GOOD)
   1244                 camtape_process_errors(softc, rc, msg, "READPOS", true);
   1245         else {
   1246                 pos->partition = ext_data.partition;
   1247                 pos->block = scsi_8btou64(ext_data.last_object)
   1248                 ltfsmsg(LTFS_DEBUG, 30398D, "next-block-to-xfer",
   1249                                 (unsigned long long) pos->block, 0, 0, softc->drive_serial);
   1250         }
   1251
```

--- a/src/tape_drivers/freebsd/cam/cam_tc.c
+++ b/src/tape_drivers/freebsd/cam/cam_tc.c
@@ -1244,7 +1244,7 @@ static int camtape_get_next_block_to_xfer(void *device, struct tc_position *pos)
 		camtape_process_errors(softc, rc, msg, "READPOS", true);
 	else {
 		pos->partition = ext_data.partition;
-		pos->block = scsi_8btou64(ext_data.last_object); // here on line no. 1247 there was semicolon missing --> so i fixed it
+		pos->block = scsi_8btou64(ext_data.last_object);
 		ltfsmsg(LTFS_DEBUG, 30398D, "next-block-to-xfer",
 				(unsigned long long) pos->block, 0, 0, softc->drive_serial);
 	}

--- a/src/tape_drivers/freebsd/cam/cam_tc.c
+++ b/src/tape_drivers/freebsd/cam/cam_tc.c
@@ -1244,7 +1244,7 @@ static int camtape_get_next_block_to_xfer(void *device, struct tc_position *pos)
 		camtape_process_errors(softc, rc, msg, "READPOS", true);
 	else {
 		pos->partition = ext_data.partition;
-		pos->block = scsi_8btou64(ext_data.last_object)
+		pos->block = scsi_8btou64(ext_data.last_object); // here on line no. 1247 there was semicolon missing --> so i fixed it
 		ltfsmsg(LTFS_DEBUG, 30398D, "next-block-to-xfer",
 				(unsigned long long) pos->block, 0, 0, softc->drive_serial);
 	}

